src/features/query-bar/components/filter/FilterButton.js

```diff
@@ -65,7 +64,8 @@ class FilterButton extends React.Component<Props, State> {
         this.setState({ isMenuOpen: !this.state.isMenuOpen });
     };
 
-    createCondition = (conditionID: string) => {
+    createCondition = () => {
+        const conditionID = this.generateConditionID();
```

Comment:
Could this be reduced to uniqueId? I don't see a purpose for generateConditionID.

Reply:
Yes